yarn.lock

```diff
@@ -971,7 +971,7 @@
   dependencies:
     regenerator-runtime "^0.12.0"
 
-"@babel/runtime@^7.1.2":
+"@babel/runtime@^7.1.2", "@babel/runtime@^7.4.5":
```

Comment:
This being a structural change with library versions being bumped, perhaps this PR can incorporate #37 and #21 and then we could close those two?